### ISSUE TITLE
Visual predictive check interaval/prediction interal

### ISSUE THREAD

**Issue by moulirc** [Open-Systems-Pharmacology/Forum#104]
![output_forum](https://user-images.githubusercontent.com/27323536/33690156-0b262430-db2e-11e7-9041-f3b513b76ec5.png)

Hi, 
I managed to do a simulation  for meropena, when  I did a visual predictive check the graph doesn't look good.. Please see attached picture and also have attached  the project file.
[Mero_run01.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/1537232/Mero_run01.zip)

**Comment by tobiasK2001** [comment]
Dear Moulirc,

it might look somewhat more conclusive if you are using log scale. And only simulate as long as you have data e.i. 4 h. Especially VPC and Confidence intervall plots could look less worse . But still the have gigantic unceartainty range of + - 4 log Scales. Iam not sure if this is a bug in the PI VPC tool... @msevestre could that be???
You might also think about not to use logP for fitting. In your PI LogP and TSspec are highly correlated...e.g. not identifiable. Indeed you get similar results only fitting TSspec, keeping logP as it is (e.g. use the one from drugbank) and maybe using another distribution model and charge dependend permeability as your compound is an acid as it seems.
But still even using only TSspec for fitting the VPC and confidance intervall looks horrible... (see attached file) 
Maybe it could be good to use more data...
Hope this might help, Tobias
[Mero_run01_edit tk.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/1539910/Mero_run01_edit.tk.zip)

**Comment by msevestre** [comment]
@moulirc Those are some weird looking graphs indeed. 
@tobiasK2001 A bug is always possible... we are only humans after all :)

**Comment by msevestre** [comment]
@moulirc  I agree with @tobiasK2001 analyses. Variation of Lipophilicity in such as huge range (-10 to 10) is not a good idea. 
The correlation matrix shows that LogP and TSSpec are highly correlated thus rendering the results of your PI somewhat useless. 
Moreover even when removing LogP from the PI, the error remains constant and the confidence interval are huge
  
![image](https://user-images.githubusercontent.com/1041237/33779858-9815fcdc-dc1c-11e7-81e0-7d0939046cc5.png)

All of which is a sign that those parameters have very little influence on your simulation. 
In other words, those graphs are exactly displaying what they should. A very large uncertainty on those parameters.

**Comment by moulirc** [comment]
@tobiasK2001 /@msevestre,

Please find attached the project file. Thanks for helping me out on this. 
As I am bit naïve to PBPK still learning.  I reduced the 
Thanks heaps for your help. 
![mero](https://user-images.githubusercontent.com/27323536/33854061-114c10ae-df0c-11e7-8693-7e8e33a86883.jpg)


[Mero_run2_corrected logp.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/1549412/Mero_run2_corrected.logp.zip)

**Comment by msevestre** [comment]
What did you change? And what else do you need to know?

**Comment by moulirc** [comment]
oops sorry forgot to complete the sentence. I reduced the start value in PI and restarted the whole work as a new project and it worked.

kind regards
Mouli
